OpenRA.Mods.Common/InstallUtils.cs

```diff
@@ -39,6 +39,9 @@ public class ContentInstaller : IGlobalModData
 		public readonly string MusicPackageMirrorList = null;
 		public readonly int ShippedSoundtracks = 0;
 
+		public readonly int[] IscfFileIds = { };
```

Comment:
Could you please add a comment above here mentioning that these are for extracting the mod-specific files from the combined TFD disk?